src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -480,6 +480,21 @@ private void VerifyReturnTypeMatch()
             _errors.AddRange(ftErrors);
         }
 
+        /// <summary>
+        /// Compute the dependencies. Called after binding. 
+        /// </summary>
+        public DependencyInfo ApplyDependencyInfoScan()

```

Comment:
>ApplyDependencyInfoScan [](http://example.com/codeflow?start=30&length=23)

Can we [Obsolete("Preview"] this. I think we'll want to make some changes.  Like moving away from strings.